util/all_tests.go

```diff
@@ -196,6 +196,12 @@ func runTestOnce(test test, mallocNumToFail int64) (passed bool, err error) {
 		cmd.Env = append(cmd.Env, "_MALLOC_CHECK=1")
 	}
 
+	if test.SkipMissing != "" {
```

Comment:
> I could potentially use a preprocess "guard" so that on specific platforms/configurations the executable is little more than a printf("PASS\n"), but that seems more problematic to me.

That's the pattern currently used (e.g. https://github.com/aws/aws-lc/blob/main/crypto/fipsmodule/rand/urandom_test.cc). IMO should follow that?